test/integration/modules/OrderModule.test.ts

```diff
@@ -357,6 +357,90 @@ describe('OrderModule', () => {
         `PermissionDenied("${trader1.accountId}", "${permission}", "${signerAddress}")`
       );
     });
+
+    it('should revert when an existing position can be liquidated (but not flagged)', async () => {
+      const { PerpMarketProxy } = systems();
+
+      const orderSide = genSide();
+      const { trader, market, marketId, collateral, collateralDepositAmount } = await depositMargin(bs, genTrader(bs));
+      const order1 = await genOrder(bs, market, collateral, collateralDepositAmount, {
+        desiredLeverage: 10,
+        desiredSide: orderSide,
+      });
+
+      await commitAndSettle(bs, marketId, trader, order1);
+
+      // Price falls/rises between 10% should results in a healthFactor of < 1.
+      //
+      // Whether it goes up or down depends on the side of the order.
+      const newMarketOraclePrice = wei(order1.oraclePrice)
+        .mul(orderSide === 1 ? 0.9 : 1.1)
+        .toBN();
+      await market.aggregator().mockSetCurrentPrice(newMarketOraclePrice);
+
+      const { healthFactor } = await PerpMarketProxy.getPositionDigest(trader.accountId, marketId);
+      assertBn.lte(healthFactor, bn(1));
+
+      // Modify the order (either +/- by 1%)
+      const order2 = await genOrder(bs, market, collateral, collateralDepositAmount, {
+        desiredSize: wei(order1.sizeDelta).mul(1.01).toBN(),
+      });
+      await assertRevert(
+        PerpMarketProxy.connect(trader.signer).commitOrder(
+          trader.accountId,
+          marketId,
+          order2.sizeDelta,
+          order2.limitPrice,
+          order2.keeperFeeBufferUsd
+        ),
+        'CanLiquidatePosition()'
+      );
+    });
+
+    it('should revert when an existing position is flagged for liquidation', async () => {
+      const { PerpMarketProxy } = systems();
+
+      const orderSide = genSide();
+      const { trader, market, marketId, collateral, collateralDepositAmount } = await depositMargin(bs, genTrader(bs));
+      const order1 = await genOrder(bs, market, collateral, collateralDepositAmount, {
+        desiredLeverage: 10,
+        desiredSide: orderSide,
+      });
+
+      await commitAndSettle(bs, marketId, trader, order1);
+
+      // Price falls/rises between 10% should results in a healthFactor of < 1.
+      //
+      // Whether it goes up or down depends on the side of the order.
+      const newMarketOraclePrice = wei(order1.oraclePrice)
+        .mul(orderSide === 1 ? 0.9 : 1.1)
+        .toBN();
+      await market.aggregator().mockSetCurrentPrice(newMarketOraclePrice);
+
+      const { healthFactor } = await PerpMarketProxy.getPositionDigest(trader.accountId, marketId);
+      assertBn.lte(healthFactor, bn(1));
```

Comment:
```suggestion
      assertBn.lt(healthFactor, bn(1));
```
This should always be `lt`, otherwise the flag on the next line might fail. But 10x liverage with 10% change +fees should mean `lt`

Reply:
N/A